CentOS_7/job/task.sh

```diff
@@ -7,6 +7,11 @@ symlink_build_dir() {
   ln -s $BUILD_DIR /build
 }
 
+# Adding this to support legacy genExec PEM key path
+copy_pem_keys() {
+  cp $BUILD_SECRETS_DIR/*.pem /tmp
```

Comment:
This _can_ fail if the user adds a directory or a file without permission to /build/secrets and runs another task after that. Allow failure?